test/integration/controlling_test.exs

```diff
@@ -80,6 +80,18 @@ defmodule Oban.Integration.ControllingTest do
         refute supervised_queue?(name2, "alpha")
       end)
     end
+
+    test "trying to stop queues that don't exist" do
+      name = start_supervised_oban!(queues: [])
+
+      # It is only possible to validate if queues exist if `local_only: true` is
+      # provided
+      assert :ok = Oban.pause_queue(name, queue: :alpha)
+
+      assert_raise ArgumentError, fn ->
+        Oban.pause_queue(name, queue: :alpha, local_only: true)
+      end
```

Comment:
With `assert_raise/3` we don't need the comment to explain it:

```suggestion
      assert :ok = Oban.pause_queue(name, queue: :alpha)

      assert_raise ArgumentError, "queue :alpha does not exist locally", fn ->
        Oban.pause_queue(name, queue: :alpha, local_only: true)
      end
```

Reply:
🤯 I never knew this was a thing, thanks so much!